controllers/medio_transporteController.js

```diff
@@ -0,0 +1,42 @@
+// controllers/medio_trasporteController.js
+const { Medio_Transporte } = require('../models');
+
+// Obtener todos los usuarios
+exports.getAllTransporte = async (req, res) => {
+    try {
+        const transportes = await Medio_Transporte.findAll();
+        res.status(200).json(transportes);
+    } catch (error) {
+        res.status(500).json({ error: 'Error al obtener los transportes' });
+    }
+};
+
+
+
+// Crear un nuevo transporte
+exports.createTransporte = async (req, res) => {
+    try {
+      const nuevoTransporte = await Medio_Transporte.create(req.body);
```

Comment:
Si medio de transporte es modelo yo definiria asi:
```suggestion
      const nuevoTransporte = await MedioTransporte.create(req.body);
```